src/analysis/classic/paladin/holy/modules/checklist/Component.tsx

```diff
@@ -30,9 +29,9 @@ const HealerChecklist = ({ thresholds, castEfficiency, combatant }: ChecklistPro
             usually something you can do to contribute to the raid. You can reduce your downtime by
             reducing the delay between casting spells, anticipating movement, moving during the GCD,
             and{' '}
-            <TooltipElement content="You can ignore this while learning to heal, but contributing DPS while healing is a major part of becoming a better than average player.">
+            {/* <TooltipElement content="You can ignore this while learning to heal, but contributing DPS while healing is a major part of becoming a better than average player.">
               when you're not healing try to contribute some damage.*
-            </TooltipElement>
+            </TooltipElement> */}
```

Comment:
Did you mean to change this file (Classic)?
If you did, it should be fine since this spec still needs to be updated for Cata.